### PR TITLE
pull request for calculator.js and countdown.js

### PR DESCRIPTION
**Pull request by chanjessica**

### CODE REVIEW

**Comment by chanjessica** [comment]
this pull-request is for Lesson3 assignment 1, 2 and 3

**Comment by chanjessica** [comment]
bonus

File: Lesson03/excercise-one.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+let myArray=['JavaScript','arrays','are','used','to','store','multiple','values','in','a','single','variable'];
+console.log(myArray);
+
+myArray.push(".");
+console.log(myArray);
+
+myArray.splice(2,1);
```
Comment (jimmynono) [inline Lesson03/excercise-one.js]: Yes! Perfect. A space after the comma would be nice, but great job at using zero-indexing.

File: Lesson03/excercise-three.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/*
+   Create an array has at least 5 strings:
+	Use a for loop to console log all of the elements plus the index and a comma
+*/
+
+let myArray=['Lesson','03','Assighment','3'];
+
+for(i=0; i<myArray.length; i++){
```
Comment (jimmynono) [inline Lesson03/excercise-three.js]: your formatting is a little wonky.

```
for (i = 0; i < myArray.length; i++) {
  console.log(i);
}
```

Comment (chanjessica) [reply Lesson03/excercise-three.js]: is space required between the operator?  i don't believed you mention in the video

Comment (chanjessica) [reply Lesson03/excercise-three.js]: i think having space between the operator and ;  is for visual.... not sure it's required??

Comment (jimmynono) [reply Lesson03/excercise-three.js]: It's not required, per se, but it is good practice and will make your code more professional. Consider it to be good "penmanship" Javascript ignores all the whitespace, but laying it out nicely is encouraged, and I'll keep pointing it out so y'all get good at it.

**Comment by chanjessica** [comment]
user array index instead of slice()

File: Lesson03/excercise-two.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+/*
+	Take a sentence of at least five words and a period.
+	Make an array from that sentence and console.log the third word.
+*/
+
+let two='A primitive value is a value that has no properties or methods.';
+convertToArray=two.split(' ');
```
Comment (jimmynono) [inline Lesson03/excercise-two.js]: it is always best to declare a variable. JS will sometimes work without them, sometimes with them. Seeing as it's inconsistent, we always want to err on the side of caution.

```
let convertToArray = two.split(' ');
```

File: Lesson03/excercise-two.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+/*
+	Take a sentence of at least five words and a period.
+	Make an array from that sentence and console.log the third word.
+*/
+
+let two='A primitive value is a value that has no properties or methods.';
```
Comment (jimmynono) [inline Lesson03/excercise-two.js]: Don't forget the spacing around the assignment operator.

```
let two = 'A primitive value is a value that has not properties or methods.'';
```

**Comment by chanjessica** [comment]
Lesson05/index.html & bank.js

**Comment by chanjessica** [comment]
add a new script bankDoWhile.js

File: Lesson02/calculator-fn.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
+var a,b, result, operator, log;
```
Comment (jimmynono) [inline Lesson02/calculator-fn.js]: Do not use var. Use const or let.

File: Lesson02/calculator-fn.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+function calculator(operator){
+   if (operator == '+') {
```
Comment (jimmynono) [inline Lesson02/calculator-fn.js]: In this case use the === comparison, it is preferred about 99.99999999% of the time.

File: Lesson02/calculator.html [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+}
+function dot(){
+   document.Calculator.display.value +=".";
+   document.Calculator.display.style.textAlign="right";
+}
+function clear(){
+   document.Calculator.display.value =" ";
+}
+</script>
+<style>
```
Comment (jimmynono) [inline Lesson02/calculator.html]: I know this wasn't part of the guidelines, but don't use inline styles. It's usually preferred to use a style sheet, it keeps the code cleaner.

File: Lesson05/bank.html [hunk 8]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (jimmynono) [inline Lesson05/bank.html]: Thanks for putting this in. It's important!

File: Lesson05/bank.js [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+•	The user should receive a warning to protect against low balances.
+•	Create an alert if the user tries to withdraw an amount that would leave them with less than $300 in their account, asking them to confirm if they're sure.
+*/
+
+let total = 1234;
+const minB = 300;
+const maxD = 50000;
+let amount ;
+function action(){
+	let action=prompt('Please select your choice: \nB for Balance\nD for deposit \nW for Withdraw \nQ to Quit');
```
Comment (jimmynono) [inline Lesson05/bank.js]: nice use of new lines to make the alert easier to read

File: Lesson05/bank.js [hunk 10]
```diff
@@ -0,0 +1,117 @@
...
+	switch(action.toUpperCase()){
+		case 'B':
+			//console.log('Your balance is $' + total);
+			//alert('Your balance is $' + total);
```
Comment (jimmynono) [inline Lesson05/bank.js]: Great! You accounted for upper and lowercase!

File: Lesson05/bank.js [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+			balance();
+			break;
+		case 'D':
+			amount=prompt('Enter the amount you want to deposit');
+			if (Number(amount) > maxD){
+				maxCap();
```
Comment (jimmynono) [inline Lesson05/bank.js]: Great use of functions to separate out the work

File: Lesson05/bank.js [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+}
+function cancel(){
+	alert('Your transaction is cancelled!');
+	checkOut()
+	action();
+}
+function checkOut(){
+	var balance = document.getElementById('balance');
+	balance.textContent = '$' + total.toFixed(2);
+	balance.style.color = "magenta";
```
Comment (jimmynono) [inline Lesson05/bank.js]: it's good practicde to use single ' marks in JS

File: Lesson03/exercise-five [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+	}
+}
+
+/*
+  Create an array with two arrays inside of it 
+    The first is the four suits of cards
+    The second is the name of each card ”ace”, “one”…”king”
+*/
+const deck=[ ['Clubs' , 'Diamonds' , 'Hearts', 'Spades'],['A','2','3','4','5','6','7','8','9','10','J','Q','K']]
+suits=deck[0].length;
```
Comment (jimmynono) [inline Lesson03/exercise-five]: Assigning variables to array lengths is nice! It's not really necessary, but a good practice.

Make sure to put spaces around your operators
```
suits = deck[0].length;
```

**Pull request closed without merging**